src/current/_includes/releases/v1.0/beta-20160407.md

```diff
@@ -2,13 +2,6 @@
 
 Release Date: {{ include.release_date | date: "%B %-d, %Y" }}
 
-<h3 id="beta-20160407-binaries">Binaries</h3>
```

Comment:
It might be good to replace the content with a para saying that the version is EOL and downloads are no longer available.  We no longer have the functionality that gives an automated admonition for EOL versions.

Reply:
Thanks for the comment. I am debating this.  TBH it feels like a lot of work for potentially very little benefit.  E.g. I think the intersection of "who is downloading a <= v20.1 binary" vs "who is a paying customer" is an empty set

Reply:
Fair enough, I'm sold.